.github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md

```diff
@@ -1,17 +1,23 @@
 ---
-name: Bug report
-about: Create a report to help us improve the CoE Starter Kit
-title: ''
-labels: ''
-assignees: ''
+name: "[ALM Accelerator] Bug report"
+about: Create a report to help us improve the ALM Accelerator for Advanced Makers
```

Comment:
Change ALM Accelerator for Advanced Makers to ALM Accelerator to make it generic to both solutions

Reply:
changed in latest commit